### PR TITLE
Remove codelab and references to submodules.

### PR DESCRIPTION
**Pull request by ndmckinley**
The codelab is no longer on the website, and the submodules have been gone for a long time.  :)

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,19 +34,6 @@ effect, an issue solved in one tool will be solved for each other tool.
 
 ## Getting Started with Magic Modules
 
-We've prepared a codelab to introduce you to Magic Modules:
-
-[![Magic Modules Codelab](images/mm-codelab.png)](https://codelabs.developers.google.com/codelabs/magic-modules/index.html)
```
Comment (rileykarson) [inline README.md:39]: Can you delete this file too?

**Review by rileykarson — APPROVED** [review]
Ah, too bad that the codelab went away. Definitely not worth getting it added back.

**Comment by msgongora** [comment]
Is there any way the content could be transferred/saved as markdown in this repo. I have been tracking down a crash I had using the `maintenance_exclusion` from `container_cluster` and now that I have found and fixed it, I was planning to push a PR and everything but since this is the first time I was interested in this codelab. I'm not sure if the current docs from this repo will be enough, I'm still learning from them.

**Comment by rileykarson** [comment]
The codelab is specific to generated resources, and for Terraform GKE cluster is a handwritten resource so most of the content won't be relevant. Regardless, here's an (extremely quick and dirty) export of the codelab content to a gist if you're interested: https://gist.github.com/rileykarson/5afc4ff7dc9a8e0971242b1b9f1452b7. Notably, code blocks ended up centered & images didn't make the transition at all.

https://gist.github.com/rileykarson/5afc4ff7dc9a8e0971242b1b9f1452b7#handwritten-files particularly may be of interest to you.

**Comment by msgongora** [comment]
appreciate it @rileykarson

**Pull request merged**